src/presolve.jl

```diff
@@ -723,3 +728,33 @@ function remove_dominated_columns!(ps::PresolveData{T}) where {T}
     end
     return nothing
 end
+
+"""
+    coefficient_strengthening!(ps::PresolveData)
+
+Perform coefficient strengthening for integer/binary variables
+on every constraints.
+
+Called once at the beginning of the presolve procedure.
+"""
+
+function coefficient_strengthening!(ps::PresolveData{T}) where {T}
+    ps.pb0.is_continuous && return nothing
+
+    zero_coefficient_strengthening!(ps)
+
+    # inactive rows
+    for (i, row) in enumerate(ps.pb0.arows)
+        if all(row.nzval .== 0)
+            ps.rowflag[i] = false
+        end
+    end
+    # inactive cols
+    for (j, col) in enumerate(ps.pb0.acols)
+        if all(col.nzval .== 0)
+            ps.colflag[j] = false
+        end
+    end
```

Comment:
Empty rows and columns need to be checked for infeasibility/unboundedness before being removed (e.g. an empty row with lower bound `1` means the problem is infeasible).

This is performed by, e.g., `EmptyRow` and `EmptyColumn` reductions, so I would rather leave it to them.
```suggestion
```